portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/asset/assetpublisher/AssetPublisherUseCase.testcase

```diff
@@ -1404,6 +1404,69 @@ definition {
 		}
 	}
 
+	@description = "LPS-103624 - URLs are not well formed when changing language in classic Display Pages."
+	@priority = "4"
+	test ViewChangeLanguageInAssetPublisherWithFriendlyURL {
```

Comment:
@Jayda-4145 since this test case is already in the `AssetPublisherUseCase.testcase` file, we don't need to mention "AssetPublisher" in the title.  Please update the test name to be `ViewInContextTranslatedWCWithFriendlyURL` - thanks!